Assets/Scripts/ProjController.cs

```diff
@@ -0,0 +1,66 @@
+using UnityEngine;
+
+public class ProjController : MonoBehaviour
+{
+    [SerializeField]
+    private int projDamage = 100;
+    [SerializeField]
+    private float projMoveSpeed = 7f;
+    [SerializeField]
+    private bool wantDeviation = true;
+    [SerializeField]
+    private float projDeviation = 1f;
+
+    public string target = "null"; // Needs to be set on creation by creating object, "Player" to damage players, "Enemy" to damage enemies
+    public Transform targetPos; // Needs to be set on creation by creating object
+
+    private Rigidbody2D rb;
+    private Vector2 moveDirect; // Movement Direction
```

Comment:
`rb` and `moveDirect` can be local variables in the `Start` method. Also, I think `moveDirect` should be renamed to `velocity`.